connectors/discord.mdx

```diff
@@ -0,0 +1,28 @@
+---
+title: Discord Connector
+description: 'Access knowledge from your Messages'
+---
+
+## How it works
+The Discord connector indexes all channels for all servers(guilds) mentioned.
+
+## Setting up
+### Authorization
+1. Login to your discord's developer portal https://discord.com/developers/applications
+2. Create a new application
+3. Head over to the `Installation` tab:
```

Comment:
Need to add adding 'bot' to scopes in the installation tab and then 'read message history' to permissions

Reply:
this is not actually necessary after testing

Reply:
We do need `read message history`. I have added a point for permissions.